src/components/Strategies/strategyData.tsx

```diff
@@ -0,0 +1,45 @@
+import { IStrategies } from '../../types';
+export const strategies: IStrategies[] = [
+  {
+    title: 'Bridgeworld (Atlas Mine)',
+    imageSrc: '../img/atlas-mine.jpg',
+    displayComingSoonText: false,
```

Comment:
Let's just shorten this to `comingSoon`. A good descriptive name is as short as possible - coming up with a good short name is very hard and you might be tempted to useEntiteSentencesToNameStuff but the longer it is the harder it is to comprehend